### PR TITLE
website detailed aid package changes

### LINKED ISSUE #235: Website detailed aid package changes
On this page : https://donor-portal-medicinesfor-lk.vercel.app/package/9 

- [ ] Insert the footer

Detail aid package

### PR DESCRIPTION
**Pull request by timashan**
## Purpose

The purpose of this PR is to fix #235

## Goals

upgrade website detailed aid package to new design
added Image
fixed progressbar overflow

### Screenshots

screencapture-localhost-3002-package-1-2022-07-26-18_16_13

### CODE REVIEW

File: donor-portal/src/pages/package/styles.css [hunk 1]
```diff
@@ -14,28 +17,39 @@
...
+
 .aid-package-description-ribbon {
   max-width: 100%;
-  height: 3rem;
-  font-size: 2rem;
+  font-size: 2.6rem;
   font-weight: bold;
   padding-left: 2rem;
-  padding-top: 0.4rem;
+  /* padding-top: 0.4rem; */
```
Comment (anjula-sack) [inline donor-portal/src/pages/package/styles.css:33]: Let's remove this

Comment (timashan) [reply donor-portal/src/pages/package/styles.css]: yep 4got tht thnx

**Comment by jayasanka-sack** [comment]
@timashan Could you please add some screenshots of your implementation to the description?

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done @timashan!

**Pull request merged**